### PR TITLE
adding jsnlog

### PR DESCRIPTION
**Pull request by mperdeck**

### CODE REVIEW

**Comment by vvakame** [comment]
*jsnlog/jsnlog.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/jsnlog
  * http://bower.io/search/?q=jsnlog
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (jsnlog/jsnlog-tests.ts or jsnlog/jsnlog-tests.tsx)
* [ ] pass the Travis-CI test?

File: jsnlog/jsnlog.d.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+// Definitions by: Mattijs Perdeck <https://github.com/mperdeck>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+// -------------------------------
+// Full documentation is at 
+// http://jsnlog.com
+// -------------------------------
+
+﻿/**
+* Copyright 2015 Mattijs Perdeck.
```
Comment (vvakame) [inline jsnlog/jsnlog.d.ts]: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE
this repository is under MIT license.

**Comment by mperdeck** [comment]
I changed the license for the files (and the entire project) to MIT, so it is in sync with DefinitelyTyped.

**Comment by vvakame** [comment]
thanks mate.
We may make changes if necessary!

**Pull request merged**